### PR TITLE
social media previews

### PR DESCRIPTION
**Pull request by stanleypliu**
~~Set the Twitter card to use 'summary_large_image' and also use 4096 x 4096 (the maximum size) variants in case images are too large for Twitter~~

EDIT: After setting the staging S3 bucket to public-read, setting the Content-Type of the images and resizing the images, the S3 hosted hero images still cannot be loaded by the web crawlers used by the social media bots. I've resorted to using the default social image for all pages which can be shared - which as it turns out still doesn't work on the news pages for unknown reasons (the GL page and other thematic area pages work). There are still useful changes I've made so this should still be merged in. 

Also, apologies for the messy commit history, tried to amend commits after pushing (instead of before)

### CODE REVIEW

**Comment by Levia** [comment]
@stacytalbot @stanleypliu What have we decided to do with this?
I can see there a few commented bits and TODOs (also in the case statement with regards the image)

**Comment by stanleypliu** [comment]
can this be merged in? it'll return previews for *most* pages with the standard `social.png` as the image
aim to return to this when there's more budget to investigate why it's not rendering the cards for news pages - no longer convinced it's just to do with the bucket as I've commented out the code to render the custom image (the hero image if present)

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -22,22 +22,31 @@ def admin_path?
...
+      'title': t('meta.site.title'),
+      'description': t('meta.site.description'),
+      'url': request.url,
+      'type': 'website',
+      'image': URI.join(root_url, helpers.image_path(t('meta.image'))),
+      'image:alt': t('meta.image_alt'),
+      'image:height': t('meta.image_height'),
+      'image:width': t('meta.image_width'),
+      'locale': 'en_GB',
+      'locale:alternate': "#{I18n.locale}_#{I18n.locale.upcase}"
```
Comment (Levia) [inline app/controllers/application_controller.rb:40]: Why are we using an hardcoded string for locale while using I18n for locale:alternate?

Comment (stanleypliu) [reply app/controllers/application_controller.rb]: sorry, this line was later removed

**Comment by Levia** [comment]
Merging with the caveat that this needs further fixes about showing the desired preview image

**Pull request merged**